providers/aws/services/vpc/vpc_endpoint_connections_trust_boundaries/vpc_endpoint_connections_trust_boundaries.py

```diff
@@ -0,0 +1,45 @@
+from config.config import get_config_var
+from lib.check.models import Check, Check_Report
+from providers.aws.services.vpc.vpc_client import vpc_client
+
+
+class vpc_endpoint_connections_trust_boundaries(Check):
+    def execute(self):
+        findings = []
+        # Get trusted account_ids from config.yaml
+        trusted_account_ids = get_config_var("trusted_account_ids")
```

Comment:
What happens if this call fails?

I think you have to handle exceptions in these functions
```
def change_config_var(variable, value):
    with open(config_yaml) as f:
        doc = yaml.safe_load(f)

    doc[variable] = value

    with open(config_yaml, "w") as f:
        yaml.dump(doc, f)


def get_config_var(variable):
    with open(config_yaml) as f:
        doc = yaml.safe_load(f)

    return doc[variable]
```

Reply:
Good catch, functions were added.